File: Workshop herald/Week 2/Java/Division.java
import java.util.*;
public class Division{
        public static void main(String[] args){
            
            Scanner input = new Scanner(System.in);
            System.out.print("Enter a number :" );
            int number1 = input.nextInt();
            
            System.out.print("Enter another number : ");
            int number2 = input.nextInt();

            int div = number1 / number2;

            System.out.printf("Division of a and b is : %.2f \n", div);
        
            
        }
}

File: Workshop herald/Week 2/Python/Division.py
num1=float(input("Enter first number"))
num2=float(input("Enter second number"))
div = num1/num2
print("The division with only two decimal is %0.2f" %div)
print("The division with scientific notation is %0.2e" %div)
print("The division with scientific notation is {:0.2e}".format(div))
